testingutils/src/test/java/com/github/awsjavakit/testingutils/networking/WiremockBugTest.java

```diff
@@ -0,0 +1,96 @@
+package com.github.awsjavakit.testingutils.networking;
```

Comment:
delete the class. Add a todo in the constant "NON_NULL_EMPTY_BODY". I would suggest to delete the class in an own commit so that it is easier to find it later